lib/mirage_block_partition.ml

```diff
@@ -79,6 +79,13 @@ module Make(B : Mirage_block.S) = struct
     let sector_start = 0L in
     partition b ~sector_size ~sector_start ~sector_end ~first_sectors
 
+  type connect' = {
+    offset: int;
+    length: int;
+    partition b ~sector_size ~sector_start ~sector_end ~first_sectors
+  }
```

Comment:
`connect'` needs to be a function. What you have here is a type definition (and not completely valid OCaml syntax). You should be able to run `dune build` to see if it compiles.

Reply:
ooh sorry my bad....i thought was asked to declare a type...I will modify that asap.

![image](https://user-images.githubusercontent.com/37302024/228784037-520e0fd1-ed7e-403b-94b6-ac280a3591bd.png)

I think this function in my screenshot (vscode) is valid right?

Reply:
That is a function. However, if you try to compile it you will discover that the variables `sector_size`, `sector_start`, `sector_end` and `first_sectors` are unbound.